consensus/polybft/fsm.go

```diff
@@ -232,7 +232,7 @@ func (f *fsm) createBridgeCommitmentTx() (*types.Transaction, error) {
 		return nil, fmt.Errorf("failed to encode input data for bridge commitment registration: %w", err)
 	}
 
-	return createStateTransactionWithData(contracts.StateReceiverContract, inputData), nil
+	return createStateTransactionWithData(f.parent.Number+1, contracts.StateReceiverContract, inputData), nil
```

Comment:
Optional nitpick (possible _slightly_ more readable)

```suggestion
	return createStateTransactionWithData(f.Height(), contracts.StateReceiverContract, inputData), nil
```